pages/feedback.js

```diff
@@ -0,0 +1,88 @@
+import React, { Component } from "react";
+import Header from "../components/typography/header";
+import withI18N from "../lib/i18nHOC";
+import Layout from "../components/layout";
+import { globalTheme } from "../theme";
+import Container from "../components/container";
+import { css } from "emotion";
+import RadioSelector from "../components/radio_selector";
+import { connect } from "react-redux";
+import HeaderLink from "../components/header_link";
+import Button from "../components/button";
+import PropTypes from "prop-types";
+
+const padding = css`
+  padding-top: 15px;
+  padding-bottom: 15px;
+`;
+const prevButton = css`
+  margin-top: 50px;
+  margin-bottom: 15px;
+`;
+export class Feedback extends Component {
+  render() {
+    const { t, i18n, questions, store } = this.props;
+    const question = questions.filter(x => x.variable_name === "feedback")[0];
```

Comment:
Why only do one question?

Reply:
there's only one multiple choice question in the design: https://app.zeplin.io/project/5acbd659dc553ec84805afd4/screen/5c3e0c5acd24f8525cbde78c